server/config.go

```diff
@@ -246,16 +246,19 @@ func (cfg *Config) Check() error {
 		return fmt.Errorf("max blob length is 0")
 	}
 
-	if cfg.SvcManagerAddr != "" && cfg.EthRPC == "" {
-		return fmt.Errorf("svc manager address is set, but Eth RPC is not set")
-	}
+	// memstore not enabled means we use eigenda as a backend, which requires these fields to be set
+	if !cfg.MemstoreEnabled {
+		if cfg.ClientConfig.RPC == "" {
+			return fmt.Errorf("eigenda disperser rpc url is not set")
+		}
 
-	if cfg.EthRPC != "" && cfg.SvcManagerAddr == "" {
-		return fmt.Errorf("eth rpc is set, but svc manager address is not set")
-	}
+		if cfg.EthRPC == "" {
+			return fmt.Errorf("eth rpc is not set")
+		}
 
-	if cfg.EthConfirmationDepth >= 0 && (cfg.SvcManagerAddr == "" || cfg.EthRPC == "") {
-		return fmt.Errorf("eth confirmation depth is set for certificate verification, but Eth RPC or SvcManagerAddr is not set")
+		if cfg.SvcManagerAddr == "" {
+			return fmt.Errorf("svc manager address is not set")
+		}
```

Comment:
currently cert verification is an optional feature but this logic makes it mandatory

Reply:
Added a `eigenda-cert-verification-enabled` flag in [6befa7c](https://github.com/Layr-Labs/eigenda-proxy/pull/138/commits/6befa7c9af47c1e8a9795372217668444f898989)
Please make sure the changes make sense.